File: Java/프로그래머스/0/120897. 약수 구하기/약수 구하기.java
class Solution {
    public int[] solution(int n) {
        int c=0;
        int m=0;
        for(int i=1; i<=n;i++){
            if(n%i==0) c++;
        }
        int a[]=new int[c];
        for(int i=1; i<=n; i++){
            if(n%i==0) {a[m]=i; m++;}
        }
        int[] answer = a;
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120897. 약수 구하기/약수 구하기.js
function solution(n) {
    var count = 0;
    for(var i=1; i<=n; i++){
        if(n%i==0) {
            count++;
        }
    }
    var answer = new Array(count);
    var m = 0;
    for(var i=1; i<=n; i++){
        if(n%i==0) {
            answer[m]=i;
            m++;
        }
    }
    return answer;
}